mav_msgs/include/mav_msgs/eigen_mav_msgs.h

```diff
@@ -161,15 +179,29 @@ struct EigenTrajectoryPoint {
                        const Eigen::Vector3d& _jerk,
                        const Eigen::Vector3d& _snap,
                        const Eigen::Quaterniond& _orientation,
-                       const Eigen::Vector3d& _angular_velocity)
+                       const Eigen::Vector3d& _angular_velocity,
+                       const Eigen::Vector3d& _angular_acceleration)
       : time_from_start_ns(_time_from_start_ns),
         position_W(_position),
         velocity_W(_velocity),
         acceleration_W(_acceleration),
         jerk_W(_jerk),
         snap_W(_snap),
         orientation_W_B(_orientation),
-        angular_velocity_W(_angular_velocity) {}
+        angular_velocity_W(_angular_velocity),
+        angular_acceleration_W(_angular_acceleration) {}
+
+  EigenTrajectoryPoint(int64_t _time_from_start_ns,
```

Comment:
Can't we overload `EigenTrajectoryPoint` to set snap and angular acceleration to zero by default? then we don't break existing code and you can use the new one whenever needed.